### ISSUE TITLE
Equal linear

### ISSUE THREAD

**Issue by diaodeyi** [rosinality/stylegan2-pytorch#212]
Hi, excuse me. What's the  exact meaning of the Equalinear? Why need to scale the Weight and bias

**Comment by rosinality** [comment]
It is equalized learning rate trick of progressivegan/stylegan(2). You can refer to this: https://arxiv.org/abs/1710.10196.
